src/EnergyPlus/SetPointManager.hh

```diff
@@ -1009,6 +905,27 @@ namespace SetPointManager {
         void calculate(EnergyPlusData &state);
     };
 
+    struct DefineSysNodeResetSetPointManager : SPBase // Derived type for System Node Reset Setpoint Manager Data
+    {
+        // Members
+        Real64 SpAtLowRef;  // Setpoint at Low Reference Temperature or Humidity Ratio (i.e., Maximum Temperature/Humidity Ratio Setpoint)
+        Real64 SpAtHighRef; // Setpoint at High Reference Temperature or Humidity Ratio (i.e., Maximum Temperature/Humidity Ratio Setpoint)
+        Real64 LowRef;      // Low Reference Temperature or Humidity Ratio
+        Real64 HighRef;     // High Reference Temperature or Humidity Ratio
+        int RefNodeNum;     // Reference Node Number
+        int NumCtrlNodes;
+        std::string CtrlNodeListName;
+        Array1D_int CtrlNodes;
+        Real64 SetPt; // current setpoint value
+
+        // Default Constructor
+        DefineSysNodeResetSetPointManager() : SpAtLowRef(0.0), SpAtHighRef(0.0), LowRef(0.0), HighRef(0.0), RefNodeNum(0), NumCtrlNodes(0), SetPt(0.0)
```

Comment:
It's preferred to drop the constructor and initialize in the declarations. (Yet another old pattern we're trying not to keep using.)

Reply:
Did not know that either. Had been seeing this new pattern here and there, but didn't realize this is something we are explicitly trying to move to.

Reply:
Hi Mike, thanks for your feedback. I'll address them all. By the way, I think I can do the same (dropping the constructor and initializing the values in the declarations) to all the structs that have this pattern, if you'd prefer that.